src/app/server/Mdns.cpp

```diff
@@ -252,11 +245,12 @@ CHIP_ERROR MdnsServer::AdvertiseOperational()
         if (fabricInfo.IsInitialized())
         {
             uint8_t mac[8];
+            chip::DeviceLayer::ConfigurationMgr().GetPrimaryMACAddress(mac);
```

Comment:
This should take a MutableByteSpan, otherwise we are shoehorning 48 and 64 bit link layer addresses in one thing. Seems unepected.

Reply:
Done